client/src/components/DatasetInformation/DatasetError.vue

```diff
@@ -8,7 +8,7 @@
             <JobDetailsProvider
                 v-if="!datasetLoading"
                 v-slot="{ result: jobDetails, loading }"
-                :jobid="dataset.creating_job"
+                :jobId="dataset.creating_job"
```

Comment:
Not critical, but now we can also write `job-id` instead, right? Thank you for adding the Mixins.